spec.html

```diff
@@ -31757,7 +31778,8 @@ <h1>Array.prototype.concat ( ..._arguments_ )</h1>
                   1. Perform ? CreateDataPropertyOrThrow(_A_, ! ToString(_n_), _subElement_).
                 1. Increase _n_ by 1.
                 1. Increase _k_ by 1.
-            1. Else _E_ is added as a single item rather than spread,
+            1. Else,
+              1. Assert: _E_ is added as a single item rather than spread.
```

Comment:
A couple of these seem like `NOTE`s rather than psuedo-code assertions. @ljharb what do you think?

Reply:
good call; commented a few more cases like this.

Reply:
4998520